packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.Props.ts

```diff
@@ -96,6 +96,11 @@ export interface IDropdownProps extends React.Props<Dropdown> {
   required?: boolean;
 
   /**
+   * Max item per page
+   */

```

Comment:
when reading this comment as a consumer, I have no idea what it's supposed to do